css-four-column-layout/index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>CSS Four Column Layout</title>
+  <link rel="stylesheet" href="styles.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Barlow:wght@100&display=swap" rel="stylesheet">
+</head>
+<body>
+  <h2>How It Works</h2>
+  <section class="row">
+    <div>
```

Comment:
almost there! These divs should be the `column-forth` elements since there are four total columns and they each take up one forth of the page. and the other columns will remain as children to the overall column